examples/next_reuse.rs

```diff
@@ -40,15 +41,15 @@ impl<C: HttpClient> Middleware<C> for Doubler {
     }
 }
 
-#[runtime::main]
-async fn main() -> Result<(), surf::Exception> {
-    femme::start(log::LevelFilter::Info)?;
-    let mut res = surf::get("https://httpbin.org/get")
-        .middleware(Doubler {})
-        .await?;
-    dbg!(&res);
-    let body = res.body_bytes().await?;
-    let body = String::from_utf8_lossy(&body);
-    println!("{}", body);
-    Ok(())
+fn main() {
```

Comment:
This should probably return:
```suggestion
fn main() -> Result<(), surf::Exception> {
```

so that `unwrap` isn't needed.

Reply:
hey @yoshuawuyts , I tried this but the `unwrap` is inside the async block and I did't find any way to make that return an `Result`. :cry: 

How can I do that?